meeting-notes/2018/2018-08-13--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,151 @@
+  - Todo
+    - Helping @mkg20001 to land re-usable private keys [#1485](https://github.com/ipfs/js-ipfs/pull/1485)
+    - Identify and resolve more bottlenecks that are causing the tests to take so long
+
+
+
+Following dweb summit I've been spending some time implementing a MDNS discovery and TCP transport that uses **libdweb**:
+
+* https://github.com/alanshaw/js-libp2p-webext-mdns
+* https://github.com/alanshaw/js-libp2p-webext-tcp
+
+These can be used with the embedded js-ipfs node that runs in IPFS companion to give it basically the same powers as js-ipfs running in Node.js. Obviously this includes local network discovery of JS or Go nodes but also p2p communication without websockets. This would be so AWESOME.
+
+I think it's important that these libp2p modules exist for IPFS and I think that IPFS (the organisation) using, testing and approving libdweb will hopefully make it a little more likely to be accepted into Firefox and then maybe adopted by Chrome and others.
+
+
+
+#### js-libp2p
+https://docs.google.com/spreadsheets/d/1HTXfgR5FyPTFhsTkFPRThkeMvHvCgJOaAs7BSl_vQ_0/edit#gid=1241853194
+
+@vasco.santos (Not Attending)
+
+- Done: 
+  - service-worker-gateway: stats page with repo size friendly
+    - https://github.com/ipfs-shipyard/service-worker-gateway/pull/26
+  - js-ipns: add public key support
+    - https://github.com/ipfs/js-ipns/pull/4
+  - js.ipfs.io
+    - several PRs tested and reviewed
+  - js-ipfs
+    - IPNS Working locally ready \o/ 
+      - https://github.com/ipfs/js-ipfs/pull/1400
+    - js-ipfs repo interop
+      - https://github.com/ipfs/js-ipfs/pull/1496
+  - js-ipfs-repo
+    - Fix interoperability with go
+      - https://github.com/ipfs/js-ipfs-repo/pull/173
+  - interop
+    - Repo (should be working with the new version of js-ipfs)
+      - https://github.com/ipfs/interop/pull/29
+    - IPNS (should be working with the new version of js-ipfs)
+      - https://github.com/ipfs/interop/pull/26
+
+- Blocked: 
+  - N/A
+
+- Next:
+  - on holidays in this week and in the next one
+    - come back on the 27th
+    - possibly unreachable on the Internet until the 20th
+    - for contacting it is preferred Slack
+
+
+#### js-ipld
+https://docs.google.com/spreadsheets/d/1Lfd91hi3nFlLRS1r-FHvK2ip2Ll6ukraufCgepw43bw/edit#gid=221584476
+
+
+#### List everyone's remaining availability
+
+7 Weeks left - 35 days total (assuming a 5 day work week)
+
+Actual availability (discard holidays, confs planned and any scheduled event that will block you from focusing on the OKRs work)
+
+
+| Name   | Actual (days) | 60% |
+|--------|---------------|-----|
+| David  | 20            | 12  |
+| Alan   | 29            | 17  |
+| Hugo   |               |     |
+| Alex   | 28            | 16  |
+| Travis | 35            | 21  |
+| Jacob  | 34            | 20  |
+| Vasco  | 25            | 15  |
+| Volker | 22            | 13  |
```

Comment:
I recommend everyone checking out the recording of this call at https://youtu.be/yBzkjM1jJV8 and run through the time exercise again.

Estimate in days how much each remaining KR will take and if you need more days than what you have, consider dropping some and communicate that by putting a 0 on the mid quarter projection.
@alanshaw @hugomrdias @achingbrain @travisperson @jacobheun @vasco-santos @vmx